spec/src/main/asciidoc/architecture.adoc

```diff
@@ -59,6 +59,10 @@ than the _vendor_ one.
 
 Required base metrics are exposed under `/metrics/base`.
 
+The `base` scope is also used for metrics generated by application frameworks, when these are based on a related (MicroProfile or other) specification
+and therefore portable between different runtimes that support the relevant framework. For example, the RESTful metrics described in this specification,
+if they are implemented by a particular runtime that supports both JAX-RS and MicroProfile Metrics, they will be placed in the `base` scope.
```

Comment:
Would like to simplify this a bit.  I'm not sure what other specs (beyond MicroProfile) we would want adding to the base scope -- and might want to deal with that case if/when it comes up.  I would suggest...

The `base` scope is used for, and only for, any metrics that are defined in MicroProfile specifications. Metrics in the base scope are intended to be portable between different MicroProfile-compatible runtimes.

Reply:
So if we just replace "MicroProfile or other" with just "MicroProfile"?

Reply:
Updated per your suggestion now